### PR TITLE
Minor docs updates

### PR DESCRIPTION
**Pull request by vulcanoidlogic**

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 Page editors are a great way to provide an excellent user experience. However, to build one is often a pretty dreadful task.
 
-There're existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as modifying the user interface and its behavior, it will most definitely involve modifying the library itself.
+There are existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as modifying the user interface and its behavior, it will most definitely involve modifying the library itself.
```
Comment (azreenashah) [inline packages/core/README.md:23]: I think we can keep the original for this one

File: packages/docs/docs/overview.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ title: Overview
 
 Page editors are a great way to provide an excellent user experience. However, to build one is often a pretty dreadful task.
 
-There're existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as tweaking the user interface, it will most definitely involve modifying the library itself.
+There are existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as tweaking the user interface, it will most definitely involve modifying the library itself.
```
Comment (azreenashah) [inline packages/docs/docs/overview.md:10]: The original can be kept for this one too

**Review by azreenashah — CHANGES_REQUESTED** [review]
Looks good! Just 2 minor changes and we should be good 💯

File: packages/core/README.md (hunk 1 above)
Comment (vulcanoidlogic) [reply packages/core/README.md]: Reverted to "There're"  in https://github.com/prevwong/craft.js/pull/129/commits/b93089054b98553e523d3a0b34f91d94e8dc19d9

Thx!!!

File: packages/docs/docs/overview.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ title: Overview
 
 Page editors are a great way to provide an excellent user experience. However, to build one is often a pretty dreadful task.
 
-There're existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as tweaking the user interface, it will most definitely involve modifying the library itself.
+There're already existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as tweaking the user interface, it will most definitely involve modifying the library itself.
```
Comment (azreenashah) [inline packages/docs/docs/overview.md:10]: Seems to be inconsistent with the `README`. Can we keep the original?

**Review by azreenashah — CHANGES_REQUESTED** [review]

**Review by azreenashah — APPROVED** [review]

**Review by prevwong — APPROVED** [review]

**Pull request merged**